### ISSUE TITLE
Emacs formatter should not output multiple lines for parse errors

### ISSUE THREAD

**Issue by sds** [bbatsov/rubocop#1801]
If you are running `parser` 2.2.0.4 or 2.2.2.0, running `rubocop --format=emacs` against the following results in a single error that spans multiple lines:

```ruby
(/
   world # Some comment containing a )
/x)
```

```
~/src/test.rb:1:2: E: unmatched close parenthesis: /
   world # Some comment containing a )
/
```

I've filed https://github.com/whitequark/parser/issues/190 with the `parser` gem for the parsing problem, but it seems like Rubocop shouldn't ever have error messages that span multiple lines when using `--format=emacs`? Or are my expectations not valid here?

**Comment by jonas054** [comment]
I think they are valid and that it wouldn't hurt to remove line breaks from Parser's diagnostic messages before outputting them, just in case there are any.
